endpoints-msp-commands/src/main/scala/com/thenewmotion/ocpi/commands/CommandResponseRoute.scala

```diff
@@ -24,15 +24,11 @@ class CommandResponseRoute(
   private val CommandNameSegment = Segment.flatMap(x => CommandName.values.find(_.name == x))
 
   private[commands] def routeWithoutRh(apiUser: GlobalPartyId)(implicit executionContext: ExecutionContext) =
-    authPathPrefixGlobalPartyIdEquality(apiUser) {
```

Comment:
Why remove this line?

Reply:
There is no need to include the global party id in the URL?  Was there due to copy/paste :)

Reply:
Oh, true...